pages/app/index.tsx

```diff
@@ -23,6 +24,66 @@ export default withPageAuthRequired(() => {
     <Pane display="flex" flexWrap="wrap" justifyContent="center" 
     marginY={majorScale(4)}
     paddingX={majorScale(3)}>
+       <Pane 
+            display="flex"
+            borderRadius={8}
+            elevation={1} 
+            background="white" 
+            marginX={20} 
+            maxWidth="800px" 
+            height="400px"
+            width="100%"
+            paddingX="10px"
+            paddingTop={majorScale(5)}
+            paddingBottom={majorScale(7)}>
+              <div style={{width:"100%", display:"flex", justifyContent:"space-between"}}> 
```

Comment:
Use `<Pane>` instead of divs. For example, this would have been:
`<Pane width="100%" display="flex" justifyContent="space-between">...</Pane>`
which is just much cleaner.